taccsite_cms/templates/guides/portal_technology.html

```diff
@@ -45,115 +49,118 @@ <h2>High-level Architecture</h2>
 
   <dl class="s-inline-dl  u-nested-text-content">
-    <dt>App.json</dt><dd>JSON file with application definitions and parameters such as (name, version, label).</dd>
-    <dt>Wrapper.sh</dt><dd>wrapper file, shell script which executes the job on HPC/Docker.</dd>
-    <dt>Test.sh</dt><dd>runs a test before the job is launched and cleans up the files.</dd>
+    <dt>App.json</dt><dd>A JSON file with application definitions and input parameters (i.e. - name, version, label).</dd>
+    <dt>Wrapper.sh</dt><dd>A shell script which executes the job on the targeted resource (i.e. - an HPC system or virtualized container).</dd>
+    <dt>Test.sh</dt><dd>A test script that is run before the job is launched to validate job inputs and parameters.</dd>
   </dl>
 
   <h3>Notifications</h3>
 
-  <p>Notification Bell enables the user to view information and status of submitted jobs.</p>
+  <p>The dashboard includes a notification icon (to the right of the History menu link) that indicates the number of unviewed notifications the use has. Selecting the history area enables the user to view the status and detailed information about all submitted jobs.</p>
 
   <h3>Search</h3>
 
-  <p>CEP has a multi-tenant capable full-text search engine with an HTTP web interface and schema-free JSON documents based on ElasticSearch. Users can search for data files, or text.</p>
+  <p>CEP has a multi-tenant capable full-text search engine based on ElasticSearch and Google. Users can search for data files or text that appears on static content pages in the CMS or within the names of files that are stored on the available storage systems for the given user.</p>
 
   <h2>Environment</h2>
 
   <p>The Core portal utilizes a wide variety of technologies developed by multiple technology vendors. Below is a list of the primary libraries, frameworks and APIs leveraged in the core portal tech stack.</p>
 
   <dl class="s-inline-dl">
+    <dt>React</dt>
+      <dd>The JavaScript framework providing all client-side functionality. This is the broswer-based client the user interacts with when accessing the portal.</dd>
+
     <dt>NginX</dt>
-      <dd>When user sends http request to call the CEP URL, it goes to nginx webserver, a proxy, intermediary for requests from clients seeking resources from other&nbsp;servers. Nginx carry the request to the Web Server Gateway.</dd>
+      <dd>The nginx webserver is as a proxy for all portal users, acting as intermediary for requests from clients seeking resources from the server. Nginx curries the request to the appropriate backend server.</dd>
```

Comment:
Do you mean "Nginx carries the request" or "Nginx proxies the request", instead of "… curries …"?

Reply:
Was using the generic courier definition (but is confusing with there being a concept of code currying) so I'll make it `routes` as that is more correct.

Reply:
Ah, okay. That was new term and concept for me.